src/Sylius/Bundle/UiBundle/composer.json

```diff
@@ -43,6 +43,7 @@
         "laminas/laminas-stdlib": "^3.2"
     },
     "conflict": {
+        "doctrine/annotations": "^2.0",
```

Comment:
Add this conflict to the `CONFLICTS.md` file with a proper description

Reply:
If I see it correctly, the last version of `doctrine/annotations` has been released 2 weeks ago, so why has the problem only now appeared? 🤔

Reply:
It might be caused by caching dependencies, I've checked some of the latest build and version 1.x was used.

Reply:
https://github.com/Sylius/Sylius/pull/14704